concourse/tasks/run-rspec-test.yml

```diff
@@ -0,0 +1,14 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: ((readonly_private_ecr_repo_url))
+    tag: govuk-coronavirus-business-volunteer-form-feature-tests
+params:
+  TEST_URL:
+inputs:
+  - name: govuk-coronavirus-business-volunteer-form
+run:
+  path: rails
+  dir: govuk-coronavirus-business-volunteer-form
+  args: ['rspec']
```

Comment:
If this task is purely for the feature tests, can we run `bundle exec rake spec:features`? It scopes it a bit.

Reply:
Thanks for the review @issyl0 and your comments - updated.